buddypress/blogs/blogs-loop.php

```diff
@@ -63,7 +63,7 @@
 
 <?php else : ?>
 
-	bp_nouveau_user_feedback( 'blogs-loop-none' );
+	<?php bp_nouveau_user_feedback( ‘blogs-loop-none’ ); ?>
```

Comment:
Is it correct that these are smart quotes?

Reply:
Thanks for catching this. I've fixed it in the master.

Reply:
Great!